rtgui/edit.cc

```diff
@@ -747,7 +747,294 @@ void Rectangle::drawToMOChannel(Cairo::RefPtr<Cairo::Context> &cr, unsigned shor
     }
 }
 
-EditSubscriber::EditSubscriber (EditType editType) : ID(EUID_None), editingType(editType), bufferType(BT_SINGLEPLANE_FLOAT), provider(NULL), dragging(false) {}
+void OPIcon::drivenPointToRectangle(rtengine::Coord &pos,
```

Comment:
If `pos` is never modified, could it be bound as `const Coord&` instead of `Coord&`?